backend/pkg/commons/db2/databasetest/bigtable.go

```diff
@@ -0,0 +1,38 @@
+package databasetest
+
+import (
+	"context"
+	"testing"
+
+	"cloud.google.com/go/bigtable"
+	"cloud.google.com/go/bigtable/bttest"
+	"google.golang.org/api/option"
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/credentials/insecure"
+)
+
+func NewBigTable(t testing.TB) (*bigtable.Client, *bigtable.AdminClient) {
+	srv, err := bttest.NewServer("localhost:0")
+	if err != nil {
+		t.Fatal(err)
+	}
+	ctx := context.Background()
```

Comment:
```suggestion
	ctx, cancel := context.WithTimeout(context.Background(), 30*time.Second)
        defer cancel()
```

Reply:
It's `databasetest/bigtable.go` is the sim bigtable but controlled through go. I don't really see the need for the context with timeout in that case.